internal/lock.go

```diff
@@ -150,11 +150,26 @@ func (l *Lock) Download(dir string, tags []string, notags []string, perm string)
 		return fmt.Errorf("nothing to download")
 	}
 	errorCh := make(chan error, total)
-	for _, r := range filteredResources {
+
+	var statusLine *StatusLine
+	if status {
+		statusLine, err = NewStatusLine(ctx, &filteredResources)
+		if err != nil {
+			return err // Do not continue if resource sizing failed.
```

Comment:
This would fail the entire download step right? We said we should just print a message and but continue the download in this case.